### PR TITLE
LL-3774 Add a warning banner for ongoing scam campaigns

### PR DESCRIPTION
**Pull request by juan-cortes**
image

### Type

Feature

### Context

https://ledgerhq.atlassian.net/browse/LL-3774

### Parts of the app affected / Test plan

- Test that the notice shows, test that it can be dismissed.

### CODE REVIEW

**Review by gre — APPROVED** [review]

**Comment by gre** [comment]
i'm ok with this, but it's probably the time to also have it auto fetched using the config.json. we'll rediscuss later.

File: src/renderer/components/banners/OngoingScams.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    content={{
+      message: <Trans i18nKey="banners.ongoingScams" />,
+      Icon: TriangleWarning,
+      right: (
+        <Link id="modal-migrate-accounts-button" onClick={() => openURL(urls.banners.ongoingScams)}>
+          <Trans i18nKey="common.learnMore" />
+        </Link>
+      ),
+    }}
+    bannerId={"migrate"}
```
Comment (gre) [inline src/renderer/components/banners/OngoingScams.js:30]: wrong ids

**Review by gre — CHANGES_REQUESTED** [review]
mmh check the ids. we seems to have "migrate" in the name. is it expected?

File: src/renderer/components/banners/OngoingScams.js (hunk 1 above)
Comment (juan-cortes) [reply src/renderer/components/banners/OngoingScams.js:30]: No, it's an oversight on my part, good catch

**Comment by juan-cortes** [comment]
> i'm ok with this, but it's probably the time to also have it auto fetched using the config.json. we'll rediscuss later. how does it work with the BCH banner?

There's a priority and only one is visible at a time, once you dismiss you see the next visible one. This was confirmed to be the expected behaviour with @cthepot-ledger granted it's not ideal since they might miss it but it's what we have.

**Comment by juan-cortes** [comment]
Updated the screenshot to keep ci happy

**Review by nabil-brn — APPROVED** [review]
✓ QA-ed

---
Ledger Live 2.15.0-sha.680cdd0

**Pull request merged**